R/env.R

```diff
@@ -723,14 +723,26 @@ env_has <- function(env = caller_env(), nms, inherit = FALSE) {
   map_lgl(nms, exists, envir = get_env(env), inherits = inherit)
 }
 
-#' Get an object from an environment
+#' Get or set an object in an environment
 #'
-#' `env_get()` extracts an object from an enviroment `env`. By
-#' default, it does not look in the parent environments.
+#' @description
+#'
+#' * `env_get()` extracts an object from an enviroment `env`. By
+#'   default, it does not look in the parent environments.
+#'
+#' * `env_set()` will assign or reassign a binding in `env` if
```

Comment:
`<<-` creates the binding in the current environment rather than in the global environment, so `env_set()` behaves the same when `create` is `TRUE`. Do you mean that by default it doesn't behave like `<<-`?